docker-hub/orgs.md

```diff
@@ -9,151 +9,228 @@ redirect_from:
+create an organization, you’ll see that you have a team, the owners team, with a
+single member. An organization owner can then create new teams and add members
+to an existing team using Docker IDs or email address and by selecting a team
+the user should be part of.
+
+A **Team** is a group of Docker users that belong to an organization. An
+organization can have multiple teams. When you first create an organization,
+you’ll see that you have a team, the owners team, with a single member. An
+organization owner can then create new teams and add members to an existing team
+using Docker IDs and by selecting a team the user should be part of.
+
+> **Note:**
+>
+> In Docker Hub, users cannot belong directly to an organization.
+They belong to teams within an organization. Organization owners (Admins) can
+[configure repository permissions](#configure-repository-permissions) on a per-team basis. For example, as an org owner,
+you can specify that all teams within an organization have Read and Write access
+to repositories A and B, whereas only specific teams have Admin access to those
+repositories. Org owners have full administrative access to all repositories
+within the organization.
+
+## Create an organization
+
+To create an organization:
+
+1. Sign into [Docker Hub](https://hub.docker.com/){: target="_blank"
+rel="noopener" class="_"} using your [Docker ID](../docker-id/index.md) or your email address.
+2. Select **Organizations**. This displays a list of your existing
+organizations. Click **Create Organization** to create a new organization.
+3. Choose a plan for your organization. See [Docker Pricing](https://www.docker.com/pricing/){: target="_blank" rel="noopener"
+class="_" id="dkr_docs_subscription_btl"} for details on the features offered
+in the Team and Business plan.
+4. Enter a name for your organization. This is the official, unique name for
+your organization in Docker Hub. Note that it is not possible to change the name
+of the organization after you've created it.
+
+      > The organization name cannot be the same as your Docker ID.
+
+5. Enter the name of your company. This is the full name of your company.
+This info is displayed on your organization page, and in the details of any
+public images you publish. You can update the company name anytime by navigating
+to your organization's **Settings** page. Click **Continue to Org size**.
+6. On the Organization Size page, specify the number of users (seats) you'd
+require and click **Continue to payment**.
+
+You've now created an organization. Select the newly created organization from
+the Organizations page. You'll now see that you have a team, the **owners** team
+with a single member (you).
+
+### View an organization
+
+To view an organization:
+
+1. Log into Docker Hub with a user account that is a member of any team in the
+   organization. You must be part of the **owners** team to access the
```

Comment:
I mentioned capitalizing or quotes, but **owners** emphasis can work too (might want to apply that other places "owners" is mentioned.

Reply:
"to access" -- I'd consider mentioning "to see and access", it won't be visible as a non owner if they're wondering where it is